portal-web/test/functional/com/liferay/portalweb/tests/enduser/analyticscloud/ABTestACSide.testcase

```diff
@@ -201,4 +201,73 @@ definition {
 		}
 	}
 
+	@description = "Story: LRAC-14070 | Automation ID: LRAC-14573 | Test Summary: Can publish variant from a terminated test and delete test in AC"
+    @priority = 5
+    test CanPublishVariantFromTerminatedTestAndDeleteTest {
+        task ("Edit the variant created") {
+            ABTest.editVariant(variantName = "Variant name");
+        }
+
+        task ("Add a heading to the page to diferentiate from the control") {
+            PageEditor.addFragment(
+                collectionName = "Basic Components",
+                fragmentName = "Heading");
+        }
+
+        task ("Click to save the variant") {
+            ABTest.saveVariant();
+        }
+
+        task ("Click to open the AB test panel") {
+            ABTest.openABTestPanel();
+
+            task ("Click to review and run the test") {
+                ABTest.runTest();
+            }
+        }
+
+        task ("Click to terminate the test") {
+            Button.click(button = "Terminate Test");
+
+            ABTest.confirmTerminateTest();
+        }
+
+        task ("Click to publish the variant") {
+            ABTest.publishVariant(
+				variantName = "Variant name");
+        }
+
+		task ("Assert the alert is shown") {
+			Alert.viewSuccessMessageText(successMessage = "Success:Variant name was published successfully.");
+		}
```

Comment:
Hey @manuferreira, I think you could put this alert inside the macro?

Reply:
https://github.com/liferay-ac/liferay-portal/pull/729/commits/3fe278a9bbadb64d2844b8baadb3984ae38d4873